pkg/semodule/policycoreutils/policycoreutils.go

```diff
@@ -38,11 +38,11 @@ func (smt *SEModulePcuHandler) SetAutoCommit(_ bool) {
 func (smt *SEModulePcuHandler) Install(modulePath string) error {
 	out, err := runSemodule("-X", "350", "-i", modulePath)
 	if err != nil {
-		smt.logger.Error(err, "Installing policy", "modulePath", modulePath)
+		smt.logger.Error(nil, "Installing policy", "modulePath", modulePath, "output", out)
```

Comment:
Why are we removing the error from here? It'll get lost from the stack trace

Reply:
It was not intentional, sorry. I experimented with the removing the error and forgot to revert it back.